elasticsearch/client.sls

```diff
@@ -0,0 +1,11 @@
+{%- from "elasticsearch/map.jinja" import client with context %}
+{%- if client.get('enabled', False) %}
+
+/etc/salt/minion.d/_elasticsearch.conf:
+  file.managed:
+  - source: salt://elasticsearch/files/_elasticsearch.conf
+  - template: jinja
+  - user: root
+  - group: root
```

Comment:
Out of curiosity, adding new conf files into /etc/salt/minion.d/ does not require a salt-minion restart?